src/routers/TabRouter.js

```diff
@@ -192,8 +192,16 @@ export default (
         }
       }
       if (activeTabIndex !== state.index) {
+        const routes = [...state.routes];
+        const activeRoute = routes[activeTabIndex];
+        if(!activeRoute.routes && action.params)
```

Comment:
Why are we checking `!activeRoute.routes` ? We still may want to pass params, even if the route has child routes